owners/src/types.js

```diff
@@ -42,4 +42,14 @@ let FileTreeMap;
  */
 let ReviewerFiles;
 
+/**
+ * The result of parsing OWNERS rules, along with any errors encountered.
+ *
+ * @typedef {{
+ *   rules: null|OwnersRule[],
+ *   errors: OwnersParserError[],
+ * }}
+ */
+let OwnersParserResult;
```

Comment:
This line might need a trailing ` // eslint-disable-line no-unused-vars` if the linter complains.

Reply:
It was missing from the `module.exports`; fixed.